src/pal/src/numa/numa.cpp

```diff
@@ -599,6 +597,14 @@ GetProcessAffinityMask(
             *lpProcessAffinityMask = processMask;
             *lpSystemAffinityMask = systemMask;
         }
+        else if (errno == EINVAL)
```

Comment:
As I have mentioned before, I was thinking this cannot happen. But if I am wrong, then we should use what you have in the other place (ideally extracting the dynamic allocation to a separate function and using it at both places)